mappings/net/minecraft/world/gen/feature/EndSpikeFeature.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/unmapped/C_ngmxunil net/minecraft/world/gen/feature/EndSpikeFeature
+	FIELD f_hqfnozdy SPIKES_COUNT I
```

Comment:
SPIKE_COUNT

Reply:
I still don't think it should be SPIKES_COUNT, It should be relative to the amount of spikes there are, so SPIKE_COUNT would be best